package.json

```diff
@@ -25,7 +25,10 @@
     "test:optimized": "pnpm run build:optimized && FOUNDRY_PROFILE=test-optimized forge test"
   },
   "dependencies": {
-    "@rhinestone/modulekit": "^0.4.17"
+    "@rhinestone/modulekit": "github:rhinestonewtf/modulekit#ab4ab5d382a1bab3dea1c88ecf0d050e111e2db4"
```

Comment:
why not npm and modulekit latest?

Reply:
Was testing out the via-ir fix, are we going to bump it again?

Reply:
ah yeah lets release a new version after via-ir